GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/GVFSUpgradeReminderTests.cs

```diff
@@ -60,7 +60,14 @@ public void RemindWhenUpgradeAvailable()
         public void NoReminderForLeftOverDownloads()
         {
             this.VerifyServiceRestartStopsReminder();
+
+            // Nuke the nuget config before running VerifyUpgradeVerbStopsReminder.
+            // This test should not use Nuget upgrader, because None ring config
+            // is ignored by Nuget upgrader.

```

Comment:
Could you add a bit more detail to the comment regarding why we can't use the Nuget upgrader during this test?

Specifically, it'd be good to capture why the Nuget upgrader will always find something to download.

Maybe something like:

```
// This test should not use Nuget upgrader because it will usually find an upgrade
// to download.  The "None" ring config doesn't stop the Nuget upgrader from checking
// its feed for updates, and the VFS4G binaries installed during functional test
// runs typically have a 0.X version number (meaning there will always be a newer
// version of VFS4G available to download from the feed).
```